bin/common/versions.sh

```diff
@@ -9,8 +9,8 @@ set -o errexit -o nounset
 
 export BOSH_CLI_VERSION="fcaa9c6caff58ab8da8c56481320681cdea492ee"
 export CFCLI_VERSION="6.37.0"
-export FISSILE_FLAVOR="develop"
-export FISSILE_VERSION="7.0.0+195.ga19161d"
+export FISSILE_FLAVOR="checks"
+export FISSILE_VERSION="7.0.0+210.g1aa92ee2"
```

Comment:
One character too long. `ee` not `ee2`.

Reply:
Bumped version (because of fissile changes).